interface/patient_file/summary/demographics.php

```diff
@@ -1,1687 +1,1638 @@
-}
-?>
-<html>
-
-<head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" type="text/css" href="../../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
-<style type="text/css">@import url(../../../library/dynarch_calendar.css);</style>
-<style type="text/css">
-.inactive_font_style{
-font-family:sans-serif;
-font-size:10pt;
-}
-</style>
-<script type="text/javascript" src="../../../library/textformat.js"></script>
-<script type="text/javascript" src="../../../library/dynarch_calendar.js"></script>
-<?php include_once("{$GLOBALS['srcdir']}/dynarch_calendar_en.inc.php"); ?>
-<script type="text/javascript" src="../../../library/dynarch_calendar_setup.js"></script>
-<script type="text/javascript" src="../../../library/dialog.js"></script>
-<script type="text/javascript" src="../../../library/js/jquery-1.6.4.min.js"></script>
-<script type="text/javascript" src="../../../library/js/common.js"></script>
-<script type="text/javascript" src="../../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot']; ?>/library/js/jquery.gritter.min.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot']; ?>/interface/main/messages/hoverover.js.php"></script>
-<link rel="stylesheet" type="text/css" href="<?php echo $GLOBALS['webroot']; ?>/library/css/jquery.gritter.css" />
-<script type="text/javascript" language="JavaScript">
-
- var mypcc = '<?php echo htmlspecialchars($GLOBALS['phone_country_code'],ENT_QUOTES); ?>';
- //////////
- function oldEvt(apptdate, eventid) {
-  dlgopen('<?php echo $GLOBALS["web_root"]; ?>/modules/calendar/add_edit_event.php?date=' + apptdate + '&eid=' + eventid, '_blank', 775, 375);
- }
-
- function advdirconfigure() {
-   dlgopen('advancedirectives.php', '_blank', 500, 450);
+      return ($pics);
   }
+  // Get the document ID of the first document in a specific catg.
+  function get_document_by_catg($pid,$doc_catg) {
+  
+      $result = array();
+  
+      if ($pid and $doc_catg) {
+        $result = sqlQuery("SELECT d.id, d.date, d.url FROM " .
+          "documents AS d, categories_to_documents AS cd, categories AS c " .
+          "WHERE d.foreign_id = ? " .
+          "AND cd.document_id = d.id " .
+          "AND c.id = cd.category_id " .
+          "AND c.name LIKE ? " .
+          "ORDER BY d.date DESC LIMIT 1", array($pid, $doc_catg) );
+          }
+  
+      return($result['id']);
+  }
+  
+  // Display image in 'widget style'
+  function image_widget($doc_id,$doc_catg)
+  {
+          global $pid, $web_root;
+          $docobj = new Document($doc_id);
+          $image_file = $docobj->get_url_file();
+          $extension = substr($image_file, strrpos($image_file,"."));
+          $viewable_types = array('.png','.jpg','.jpeg','.png','.bmp','.PNG','.JPG','.JPEG','.PNG','.BMP'); // image ext supported by fancybox viewer
```

Comment:
$finfo = new finfo;
 
$fileinfo = $finfo->file($file, FILEINFO_MIME);
Should use fileinfo extension